api/Window.json

```diff
@@ -4697,7 +4909,9 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "1"
+              "version_added": "1",
+              "partial_implementation": true,
+              "notes": "Behavior is inconsistent/unreliable."
```

Comment:
Should screenX have a similar note?

Reply:
Ooop, the issue here is that I've got the note on the `screenY` main feature as well as the `relative-multi-screen` subfeature, whereas it should just be on the subfeature. I've fixed that now.